TreeDB/documentservice/rag_parity_test.go

```
@@ -0,0 +1,436 @@
+func TestHTTPDenseAnnRouteParityWithExactRoute(t *testing.T) {
+	svc, db := newTestService(t)
+	defer db.Close()
+	handler := NewHandler(svc)
+	createBody := `{"name":"docs","dimension":4}`
+	status, payload := ragParityPost(t, handler, "/v1/indexes", createBody)
+	if status != http.StatusOK {
+		t.Fatalf("create index status=%d body=%s", status, mustJSON(payload))
+	}
+	docs := make([]Document, 0, 24)
+	for i := 0; i < 24; i++ {
+		content := "topic shared filler"
```

golangci-lint (2.12.2)

[error] ineffectual assignment to content

(ineffassign)